test/scripts/base-host-check.sh

```diff
@@ -80,16 +80,16 @@ get_oscap_score() {
 }
 
 check_ca_cert() {
-    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
+    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2- | tr '[:upper:]' '[:lower:]')
 
-    echo "📗 Checking CA cert anchor file"
-    if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
+    echo "📗 Checking CA cert anchor file serial $serial"
+    if ! [ -e "/etc/pki/ca-trust/source/anchors/27894af897dd2423607045716438a725f28a6d0b.pem" ]; then
```

Comment:
Please use the variable `${serial}` instead.

Reply:
Done, note the subject is hardcoded few lines below, I could not figure out an easy way of finding the filename, it is generated by `c_rehash` OpenSSL command. A bit more challenging.

Reply:
I have found a dumb way to generalize this a bit replacing spaces with underscores. To give you some context, the `update-ca-certificates` command copies the certs into the `extracted` directory under `Common_name_with_underscores.pem` and then creates some symlinks. These are not serials, it is something different. The stage does copy the file into `anchors` directory with serial in the name.